templates/download/intel-iei-tank-870.html

```diff
@@ -89,19 +76,140 @@ <h3 class="p-stepped-list__title">
             </ol>
           </div>
         </li>
-        {% with number=6 %}{% include "download/iot/_first-boot-setup.html" %}{% endwith %}
-        {% with number=7 %}{% include "download/iot/_login.html" %}{% endwith %}
+        {% include "download/shared/_first-boot-setup.html" %}
+        {% include "download/shared/_login.html" %}
       </ol>
     </div>
   </div>
 </section>
 
-{% with strip="p-strip--light" %}{% include "download/iot/_boot-tips-strip.html" %}{% endwith %}
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-3 col-medium-6 p-section--shallow">
+      <h2>Next steps</h2>
+    </div>
+    <div class="col-9 col-medium-5 col-start-medium-2">
+      <div class="row">
+        <hr class="p-rule u-hide--small u-hide--large" />
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">First boot tips</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <ol class="p-list--divided">
+            <li class="p-list__item">During setup, <code>console-conf</code> will download the SSH key registered with your Store account and configure it so you can log into the device via <code>ssh &lt;Ubuntu SSO account name&gt;@&lt;device IP address&gt;</code> without a password.</li>
+            <li class="p-list__item">There is no default <code>ubuntu</code> user on these images, but you can run <code>sudo passwd &lt;account name&gt;</code> to set a password if you need a local console login.</li>
+          </ol>
+        </div>
+      </div>
+      <hr />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Get started with snaps</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>Your board is now ready to have snaps installed, it's time to use the snap command to install your first snap.</p>
+          <p>The <a href="https://snapcraft.io/store">Snap Store</a> is where you can find the best Linux apps packaged as snaps to install on your Ubuntu device and get started with your secure IoT journey.</p>
```

Comment:
The copy doc has a link to `install your first snap`  https://docs.snapcraft.io/getting-started/3876

Reply:
I had to update all the content in the copy doc, and then asked Juan to take another look at the copy doc for these kind of inconsistency issues...

Reply:
We were told that live content is the source of truth for these pages and this matches the copy on prod, so I'll leave it as is for now